rollup/npmHelpers.js

```diff
@@ -84,6 +86,7 @@ export function makeBaseNPMConfig(options = {}) {
       removeESLintCommentsPlugin,
       removeBlankLinesPlugin,
       extractPolyfillsPlugin,
+      debugBuildStatementReplacePlugin,
```

Comment:
In this case I don't think it makes a behavioral difference, but can this go between the sucrase and constToVar plugins? (It's more reflective of the order in which things actually happen.)

Reply:
Done. 935f9b19edc4fdf2865c1c4b76b273bcc4148f2c